cli/pkg/backend/aws/k8s_compute_backend.go

```diff
@@ -184,14 +190,18 @@ func (k8s *K8SComputeBackend) WriteParam(
 	return nil
 }
 
+func (k8s *K8SComputeBackend) getDeploymentName(stackName string, serviceName string) string {
+	return fmt.Sprintf("%s-%s", stackName, serviceName)
+}
+
 func (k8s *K8SComputeBackend) PrintLogs(ctx context.Context, stackName string, serviceName string, opts ...util.PrintOption) error {
-	deploymentName := fmt.Sprintf("%s-%s", stackName, serviceName)
-	labelSelector := v1.LabelSelector{MatchLabels: map[string]string{"app": deploymentName}}
-	pods, err := k8s.ClientSet.CoreV1().Pods(k8s.HappyConfig.K8SConfig().Namespace).List(ctx, v1.ListOptions{
-		LabelSelector: labels.Set(labelSelector.MatchLabels).String(),
-	})
+	pods, err := k8s.getPods(ctx, stackName, serviceName)
+	if err != nil {
+		return errors.Wrap(err, "unable to retrieve a list of pods")
+	}
```

Comment:
error handling is duplicated
```suggestion
	pods, err := k8s.getPods(ctx, stackName, serviceName)
```